doc/releasenotes/16_0_0_summary.md

```diff
@@ -67,6 +67,24 @@ If you have code searching for error strings from Vitess, this is a breaking cha
 Many error strings have been tweaked.
 If your application is searching for specific errors, you might need to update your code.
 
+#### Logstats Table and Keyspace removed
+
+Information about which tables are used is now reported by the field TablesUsed added in v15, that is a string array, listing all tables and which keyspace they are in.
+The Table/Keyspace fields were deprecated in v15 and are now removed in the v16.0.0 release of Vitess.
+
+#### Removed Stats
+
+The stat `QueryRowCounts` is removed in V16. `QueryRowsAffected` and `QueryRowsReturned` can be used instead to gather the same information.
```

Comment:
We are being inconsistent in how we use v15 vs V16 (capitalization). This is a nit but there is another inconsistency as well - QueryRowCounts is removed in v16 but `enale_semi_sync` is removed in v16.0.0.
Can you make them all consistent?

Reply:
Sure, I'll make the change.

Reply:
Done. I have gone with lower case v and then the number like v15, v16 and v17.